prody2/viewers/viewer_gnm.py

```diff
@@ -0,0 +1,266 @@
+                      label='Display mode animation with VMD?') 
+        group.addParam('displayDistanceProfile', LabelParam, default=False,
+                      label="Plot mode distance profile?",
+                      help="Unitary shift of each atom or pseudoatom along the mode that is requested to be animated.")
+        group.addParam('displaySingleSqFluct', LabelParam, default=False,
+                      label="Plot mode square fluctuation?",
+                      help="Shows the single mode Square Fluctuation")
+        group.addParam('cumulSqFluct', BooleanParam, default=False,
+                      #condition=metric==NMA_METRIC_OVERLAP,
+                      label='Display cumulative Square Fluctuation?',
+                      help='Show the cumulative MSF of the modes selected or show the single MSF of the mode selected')
+        
+        form.addParam('displayVmd2', LabelParam,
+                      condition=os.path.isfile(nmdFile),
+                      label="Display mode animations with VMD NMWiz?",
+                      help="Use ProDy Normal Mode Wizard to view all modes in a more interactive way. "
+                           "See http://prody.csb.pitt.edu/tutorials/nmwiz_tutorial/nmwiz.html") 
+        
+        group = form.addGroup('GNM covariance and Cross-correlation')
+        group.addParam ('displayCovMatrix', LabelParam,
+                      #condition=have_matrix == True,
+                      label='Display Covariance matrix?',
+                      help='Covariance matrices are shown as heatmaps.')
+        group.addParam ('displayCrossCorrMatrix', LabelParam,
+                      #condition=have_matrix == True,
+                      label='Display Cross Correlation matrix?',
+                      help='Cross Correlation matrices are shown as heatmaps.')
+        
+
+    def _getVisualizeDict(self):
+        return {'displayModes': self._viewParam,
+                'displayMaxDistanceProfile': self._viewParam,
+                'displaySqFlucts': self._viewSQF,
+                'displayVmd': self._viewSingleMode,
+                'displayVmd2': self._viewSingleMode,
+                'displayDistanceProfile': self._viewSingleMode,
+                'displaySingleSqFluct': self._viewSQF,
+                'displayCovMatrix': self._viewMatrix,
+                'displayCrossCorrMatrix': self._viewMatrix
+                } 
+
+    def _viewMatrix(self, paramName):
+        """ visualisation for 2D covariance and Cross correlation matrices""" 
+        if paramName=='displayCovMatrix':
+            matrix = prody.parseArray(self.protocol.matrixFileCV.getFileName())
+            title = 'Covariance matrix'
+        else:
+            matrix = prody.parseArray(self.protocol.matrixFileCC.getFileName())
+            title = 'Cross Correlation matrix'
+
+        plotter = EmPlotter(mainTitle=title)
+        plot = prody.showMatrix(matrix, vmax=1, vmin=-1, origin='lower')
```

Comment:
You probably want to have separate commands for each. The non-normalised one shouldn't have vmin and vmax